src/examples/components/badges/LargeBadges.tsx

```diff
@@ -1,44 +1,47 @@
 import * as React from "react";
-import { Body, Container, Head, Html, Preview } from "@react-email/components";
+import {
+  Body,
+  Container,
+  Head,
+  Html,
+  Preview,
+  Section,
+} from "@react-email/components";
 import { Badge } from "@mailingui/components";
 
-const SmallBadges = () => (
+const LargeBadges = () => (
   <Html>
     <Head />
-    <Preview>Small Badges</Preview>
+    <Preview>Large Badges</Preview>
     <Body style={main}>
       <Container style={container}>
-        <span style={{ padding: 10 }}>
+        <Section style={{ textAlign: "center" as const }}>
           <Badge variant="default" size="lg">
             Badge
           </Badge>
-        </span>
-        <span style={{ padding: 10 }}>
+          <span style={{ padding: 10 }} />
           <Badge variant="danger" size="lg">
             Badge
           </Badge>
-        </span>
-        <span style={{ padding: 10 }}>
+          <span style={{ padding: 10 }} />
           <Badge variant="success" size="lg">
             Badge
           </Badge>
-        </span>
-        <span style={{ padding: 10 }}>
+          <span style={{ padding: 10 }} />
           <Badge variant="info" size="lg">
             Badge
           </Badge>
-        </span>
-        <span style={{ padding: 10 }}>
+          <span style={{ padding: 10 }} />
           <Badge variant="warning" size="lg">
             Badge
           </Badge>
-        </span>
+        </Section>
```

Comment:
Not against this at all. It works, but I don't if we could be a bit more consistent with this by using a [Column Primitive](https://react.email/docs/components/column) from React.email

I want us to think about this because this is very likely a pattern we will have to repeat in all or most examples and now it contains non-functional markup (the spans just for padding).

Reply:
Thanks for trying this out